components/Card/index.js

```diff
@@ -118,6 +119,10 @@ export default function Card({ initialItems, items }) {
           );
         })}
       </StyledUl>
+      <section>
+        <button onClick={deleteAllAlert}>Delete All</button>
+      </section>
+      <br />
```

Comment:
This section doesn't do anything.
And do you really need a `<br />`

Reply:
`<br/>` is there for not overlaping the navigation . what is the better way here semanticly instead of section ?

Reply:
It depends on what you wish to achieve. The button itself is a semantic element and if you only wrapped it in a section for styling, you could also go with a div.
Another option would be to leave the wrapper out entirely.

If you are wondering when to use a section:
a section should always have a heading and it is a generic stand alone section. See this as [reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section).